src/Editable/WebData.elm

```diff
@@ -0,0 +1,124 @@
+module Editable.WebData
+    exposing
+        ( EditableWebData(..)
+        , map
+        , notAskedReadOnly
+        , state
+        , toEditable
+        , toWebData
+        )
+
+{-| An EditableWebData represents an Editable value, along with WebData.
+
+`EditableWebData` is a wrapper type around [Editable](http://package.elm-lang.org/packages/stoeffel/editable/latest)
+and [WebData](http://package.elm-lang.org/packages/krisajenkins/remotedata/latest)
+
+It is used in order to keep track of the state of the Editable upon saving. That is,
+as we change teh `Editable` toEditable, and send it to the backend, we can keep track of their status
+(e.g. `RemoteData.Success` or `RemoteData.Failure`).
+
+@docs EditableWebData, notAskedReadOnly, map, toEditable, state, toWebData
+
+-}
+
+import Editable exposing (Editable(..))
+import RemoteData exposing (RemoteData(..), WebData)
+
+
+{-| A wrapper for `Editable`, that allows provides the means to track saving
+back to the backend via `WebData`.
+
+    import Editable
+
+    view : EditableWebData String -> Html msg
+    view editableWebData =
+        let
+            toEditable =
+                Editable.WebData.toEditable |> Editable.value
+
+            toWebData =
+                Editable.WebData.toWebData
+        in
+        text <| "Editable value is: " ++ toString toEditable ++ " with a WebDataValue of " ++ toString toWebData
+
+-}
+type EditableWebData a
+    = EditableWebData (Editable a) (WebData ())
+
+
+{-| Creates a new `EditableWebData`.
+-}
+notAskedReadOnly : a -> EditableWebData a
+notAskedReadOnly record =
+    EditableWebData (Editable.ReadOnly record) NotAsked
+
+
+{-| Maps function to the `Editable`.
+
+    import Editable
+
+    Editable.WebData.notAskedReadOnly "old"
+        |> Editable.WebData.map (Editable.edit)
+        |> Editable.WebData.map (Editable.update "new")
+        |> Editable.WebData.toEditable
+        |> Editable.value --> "new"
+
+-}
+map : (Editable a -> Editable b) -> EditableWebData a -> EditableWebData b
```

Comment:
Yeah, I wasn't sure if it will be clear that the `map`ing would be to an inner type (`Editable`), but I guess we can go with that.

I see you changed signature:

```diff
- (Editable a -> Editable a) -> EditableWebData a -> EditableWebData a
+ (Editable a -> Editable b) -> EditableWebData a -> EditableWebData b
```

Can you please explain? I'd expect to always be with the `a` type